src/parkinglot/canvas_animation.mixin.ts

```diff
@@ -22,10 +22,11 @@ Object.assign(StaticCanvas.prototype, {
   fxCenterObjectH: function (object, callbacks) {
     callbacks = callbacks || {};
 
-    var empty = function () {},
-      onComplete = callbacks.onComplete || empty,
-      onChange = callbacks.onChange || empty,
-      _this = this;
+    const empty = Function
+
+    const onComplete = callbacks.onComplete || empty
+    const onChange = callbacks.onChange || empty
+    const _this = this;
```

Comment:
turn the functions into arrow functions and remove this line
https://github.com/fabricjs/fabric.js/pull/8726/files#diff-6db3fd5f9757a949bf2ea70c6ea366ecb5c1e3478306a26992b4e7f490c9c4c9R36-R45